account-setup.rst

```diff
@@ -39,7 +39,7 @@ After verifying your email address, you can access the customer portals. There a
 
 * `Ubuntu Pro dashboard <Pro_dashboard_>`_
 * `Canonical Support Portal`_ 
-* `Landscape <landscape_saas_>`_ (additional set-up will be required) 
+* `Landscape <landscape_saas_>`_ (see below for the additional set-up) 
```

Comment:
I think the phrasing here doesn't quite sound right. What about:

"See below for additional set-up instructions"

@s-makin @WajeehaHamid